beacon-chain/operations/slashings/service.go

```diff
@@ -193,13 +193,5 @@ func (p *Pool) validatorSlashingPreconditionCheck(
 	if validator.ExitEpoch() < helpers.CurrentEpoch(state) {
 		return false, nil
 	}
-	// Checking if the validator has already been slashed.
```

Comment:
Why was this removed?

Reply:
slashed vector in state is not a validator ids that were slashed
it is gwei that was slashed at each epoch